ga4gh/datamodel/variants.py

```diff
@@ -75,6 +75,17 @@ def toProtocolElement(self):
         gaCallSet.sampleId = self._sampleName
         return gaCallSet
 
+    def getGaCall(self):
+        """
+        Returns a Global Alliance Call object that is suitable for this
+        CallSet.
+        """
+        gaCall = protocol.GACall()
+        gaCall.callSetId = self._id
+        gaCall.callSetName = self._sampleName
+        gaCall.sampleId = self._sampleName
+        return gaCall
+
```

Comment:
I'm not sure if this method is pulling its weight here, since its naming and placement is somewhat confusing. Its only added value is to fill out a couple of fields of a generic GACall object with the callSet info, and this could easily be accomplished below outside of a dedicated method.